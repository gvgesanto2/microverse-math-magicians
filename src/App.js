import Calculator from './components/calculator/calculator.component';

import './App.scss';

function App() {
  return (
    <div className="main-container">
      <Calculator />
    </div>
  );
}

export default App;
