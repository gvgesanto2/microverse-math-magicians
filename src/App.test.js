import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import App from './App';

describe('App component', () => {
  it('should render correctly with the Homepage content', () => {
    const component = render(
      <MemoryRouter>
        <App />
      </MemoryRouter>,
    );
    expect(component).toMatchSnapshot();
  });

  it('should render correctly with the Calculator Page content', () => {
    const calculatorRoute = '/calculator';
    const component = render(
      <MemoryRouter initialEntries={[calculatorRoute]}>
        <App />
      </MemoryRouter>,
    );
    expect(component).toMatchSnapshot();
  });

  it('should render correctly with the Quote Page content', () => {
    const quoteRoute = '/quote';
    const component = render(
      <MemoryRouter initialEntries={[quoteRoute]}>
        <App />
      </MemoryRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
