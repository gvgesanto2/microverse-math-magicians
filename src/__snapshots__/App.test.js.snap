// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly with the Calculator Page content 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <div
          class="header__content"
        >
          <a
            data-testid="logo-link"
            href="/"
          >
            <span
              class="header__logo"
            >
              math magicians
            </span>
          </a>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li>
                <a
                  class="header__nav-link"
                  href="/"
                >
                  home
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/calculator"
                >
                  calculator
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/quote"
                >
                  quote
                </a>
              </li>
            </ul>
          </nav>
          <div
            class="header__hamburger-container"
          >
            <div
              class="hamburger-menu "
            >
              <button
                aria-label="Open hamburger menu"
                class="hamburger-btn "
                type="button"
              >
                <span
                  class="hamburger-btn__icon"
                >
                   
                </span>
              </button>
              <div
                class="hamburger-menu__background"
              >
                 
              </div>
              <nav
                class="hamburger-menu__nav"
              >
                <ul
                  class="hamburger-menu__nav-list"
                >
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/"
                    >
                      <span>
                        01
                      </span>
                      home
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/calculator"
                    >
                      <span>
                        02
                      </span>
                      calculator
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/quote"
                    >
                      <span>
                        03
                      </span>
                      quote
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </header>
      <main
        class="main"
      >
        <section
          class="calculator-page"
        >
          <h1
            class="calculator-page__title"
          >
            Let's do some math!
          </h1>
          <div
            class="calculator"
          >
            <div
              class="calculator__display"
            >
              0
            </div>
            <div
              class="buttons-grid"
            >
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                AC
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                +/-
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                %
              </button>
              <button
                class="buttons-grid__btn buttons-grid__btn--primary "
                type="button"
              >
                ÷
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                7
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                8
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                9
              </button>
              <button
                class="buttons-grid__btn buttons-grid__btn--primary "
                type="button"
              >
                x
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                4
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                5
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                6
              </button>
              <button
                class="buttons-grid__btn buttons-grid__btn--primary "
                type="button"
              >
                -
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                1
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                2
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                3
              </button>
              <button
                class="buttons-grid__btn buttons-grid__btn--primary "
                type="button"
              >
                +
              </button>
              <button
                class="buttons-grid__btn  buttons-grid__btn--lg"
                type="button"
              >
                0
              </button>
              <button
                class="buttons-grid__btn  "
                type="button"
              >
                .
              </button>
              <button
                class="buttons-grid__btn buttons-grid__btn--primary "
                type="button"
              >
                =
              </button>
            </div>
          </div>
        </section>
      </main>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <div
        class="header__content"
      >
        <a
          data-testid="logo-link"
          href="/"
        >
          <span
            class="header__logo"
          >
            math magicians
          </span>
        </a>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li>
              <a
                class="header__nav-link"
                href="/"
              >
                home
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/calculator"
              >
                calculator
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/quote"
              >
                quote
              </a>
            </li>
          </ul>
        </nav>
        <div
          class="header__hamburger-container"
        >
          <div
            class="hamburger-menu "
          >
            <button
              aria-label="Open hamburger menu"
              class="hamburger-btn "
              type="button"
            >
              <span
                class="hamburger-btn__icon"
              >
                 
              </span>
            </button>
            <div
              class="hamburger-menu__background"
            >
               
            </div>
            <nav
              class="hamburger-menu__nav"
            >
              <ul
                class="hamburger-menu__nav-list"
              >
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/"
                  >
                    <span>
                      01
                    </span>
                    home
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/calculator"
                  >
                    <span>
                      02
                    </span>
                    calculator
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/quote"
                  >
                    <span>
                      03
                    </span>
                    quote
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </header>
    <main
      class="main"
    >
      <section
        class="calculator-page"
      >
        <h1
          class="calculator-page__title"
        >
          Let's do some math!
        </h1>
        <div
          class="calculator"
        >
          <div
            class="calculator__display"
          >
            0
          </div>
          <div
            class="buttons-grid"
          >
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              AC
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              +/-
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              %
            </button>
            <button
              class="buttons-grid__btn buttons-grid__btn--primary "
              type="button"
            >
              ÷
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              7
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              8
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              9
            </button>
            <button
              class="buttons-grid__btn buttons-grid__btn--primary "
              type="button"
            >
              x
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              4
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              5
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              6
            </button>
            <button
              class="buttons-grid__btn buttons-grid__btn--primary "
              type="button"
            >
              -
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              1
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              2
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              3
            </button>
            <button
              class="buttons-grid__btn buttons-grid__btn--primary "
              type="button"
            >
              +
            </button>
            <button
              class="buttons-grid__btn  buttons-grid__btn--lg"
              type="button"
            >
              0
            </button>
            <button
              class="buttons-grid__btn  "
              type="button"
            >
              .
            </button>
            <button
              class="buttons-grid__btn buttons-grid__btn--primary "
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </section>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`App component should render correctly with the Homepage content 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <div
          class="header__content"
        >
          <a
            data-testid="logo-link"
            href="/"
          >
            <span
              class="header__logo"
            >
              math magicians
            </span>
          </a>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li>
                <a
                  class="header__nav-link"
                  href="/"
                >
                  home
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/calculator"
                >
                  calculator
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/quote"
                >
                  quote
                </a>
              </li>
            </ul>
          </nav>
          <div
            class="header__hamburger-container"
          >
            <div
              class="hamburger-menu "
            >
              <button
                aria-label="Open hamburger menu"
                class="hamburger-btn "
                type="button"
              >
                <span
                  class="hamburger-btn__icon"
                >
                   
                </span>
              </button>
              <div
                class="hamburger-menu__background"
              >
                 
              </div>
              <nav
                class="hamburger-menu__nav"
              >
                <ul
                  class="hamburger-menu__nav-list"
                >
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/"
                    >
                      <span>
                        01
                      </span>
                      home
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/calculator"
                    >
                      <span>
                        02
                      </span>
                      calculator
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/quote"
                    >
                      <span>
                        03
                      </span>
                      quote
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </header>
      <main
        class="main"
      >
        <h1
          class="home__title"
        >
          Welcome to our page!
        </h1>
        <div
          class="home__text-box"
        >
          <p
            class="home__text"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos quam deserunt reiciendis libero, quia sint rem ipsa at aliquid veritatis corrupti nam, praesentium ab accusamus dolores, esse laudantium ratione perferendis! Itaque, earum odio velit harum ipsa amet repellat optio possimus! Velit animi non tempore modi veniam, itaque odio voluptates hic facilis provident mollitia at doloremque distinctio minus ad autem sit perferendis temporibus nobis quod fugit! Officia optio laudantium modi provident vel consequatur, tempore ducimus adipisci vero at. Consequatur aliquam autem temporibus dolore incidunt porro! Harum eligendi provident velit architecto doloribus veritatis exercitationem adipisci porro. Debitis, et laudantium! Officia laboriosam fugit nihil temporibus similique quas mollitia aperiam quisquam incidunt vero est cumque eveniet sit ut qui, unde repudiandae ducimus magnam sint necessitatibus eligendi amet. Odit recusandae commodi sunt illum assumenda, nostrum non, veritatis laudantium consectetur magnam qui numquam maiores facilis eveniet iste cumque dolorem ea culpa voluptatibus quas adipisci earum. Porro tenetur et soluta qui labore autem vero molestias amet?
          </p>
          <p
            class="home__text"
          >
            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cumque repellat quo ullam quaerat culpa vero vitae, sint voluptatibus saepe incidunt! Est nobis consequatur illum voluptas sed at explicabo, necessitatibus repellat blanditiis cum sapiente vero adipisci laborum eius laboriosam alias sequi aspernatur nisi. Numquam repellendus nam quos nesciunt illo, ratione sequi.
          </p>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <div
        class="header__content"
      >
        <a
          data-testid="logo-link"
          href="/"
        >
          <span
            class="header__logo"
          >
            math magicians
          </span>
        </a>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li>
              <a
                class="header__nav-link"
                href="/"
              >
                home
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/calculator"
              >
                calculator
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/quote"
              >
                quote
              </a>
            </li>
          </ul>
        </nav>
        <div
          class="header__hamburger-container"
        >
          <div
            class="hamburger-menu "
          >
            <button
              aria-label="Open hamburger menu"
              class="hamburger-btn "
              type="button"
            >
              <span
                class="hamburger-btn__icon"
              >
                 
              </span>
            </button>
            <div
              class="hamburger-menu__background"
            >
               
            </div>
            <nav
              class="hamburger-menu__nav"
            >
              <ul
                class="hamburger-menu__nav-list"
              >
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/"
                  >
                    <span>
                      01
                    </span>
                    home
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/calculator"
                  >
                    <span>
                      02
                    </span>
                    calculator
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/quote"
                  >
                    <span>
                      03
                    </span>
                    quote
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </header>
    <main
      class="main"
    >
      <h1
        class="home__title"
      >
        Welcome to our page!
      </h1>
      <div
        class="home__text-box"
      >
        <p
          class="home__text"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos quam deserunt reiciendis libero, quia sint rem ipsa at aliquid veritatis corrupti nam, praesentium ab accusamus dolores, esse laudantium ratione perferendis! Itaque, earum odio velit harum ipsa amet repellat optio possimus! Velit animi non tempore modi veniam, itaque odio voluptates hic facilis provident mollitia at doloremque distinctio minus ad autem sit perferendis temporibus nobis quod fugit! Officia optio laudantium modi provident vel consequatur, tempore ducimus adipisci vero at. Consequatur aliquam autem temporibus dolore incidunt porro! Harum eligendi provident velit architecto doloribus veritatis exercitationem adipisci porro. Debitis, et laudantium! Officia laboriosam fugit nihil temporibus similique quas mollitia aperiam quisquam incidunt vero est cumque eveniet sit ut qui, unde repudiandae ducimus magnam sint necessitatibus eligendi amet. Odit recusandae commodi sunt illum assumenda, nostrum non, veritatis laudantium consectetur magnam qui numquam maiores facilis eveniet iste cumque dolorem ea culpa voluptatibus quas adipisci earum. Porro tenetur et soluta qui labore autem vero molestias amet?
        </p>
        <p
          class="home__text"
        >
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cumque repellat quo ullam quaerat culpa vero vitae, sint voluptatibus saepe incidunt! Est nobis consequatur illum voluptas sed at explicabo, necessitatibus repellat blanditiis cum sapiente vero adipisci laborum eius laboriosam alias sequi aspernatur nisi. Numquam repellendus nam quos nesciunt illo, ratione sequi.
        </p>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`App component should render correctly with the Quote Page content 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <div
          class="header__content"
        >
          <a
            data-testid="logo-link"
            href="/"
          >
            <span
              class="header__logo"
            >
              math magicians
            </span>
          </a>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li>
                <a
                  class="header__nav-link"
                  href="/"
                >
                  home
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/calculator"
                >
                  calculator
                </a>
              </li>
              <li>
                <a
                  class="header__nav-link"
                  href="/quote"
                >
                  quote
                </a>
              </li>
            </ul>
          </nav>
          <div
            class="header__hamburger-container"
          >
            <div
              class="hamburger-menu "
            >
              <button
                aria-label="Open hamburger menu"
                class="hamburger-btn "
                type="button"
              >
                <span
                  class="hamburger-btn__icon"
                >
                   
                </span>
              </button>
              <div
                class="hamburger-menu__background"
              >
                 
              </div>
              <nav
                class="hamburger-menu__nav"
              >
                <ul
                  class="hamburger-menu__nav-list"
                >
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/"
                    >
                      <span>
                        01
                      </span>
                      home
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/calculator"
                    >
                      <span>
                        02
                      </span>
                      calculator
                    </a>
                  </li>
                  <li>
                    <a
                      class="hamburger-menu__link"
                      href="/quote"
                    >
                      <span>
                        03
                      </span>
                      quote
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </header>
      <main
        class="main"
      >
        <section
          class="quote-page"
        >
          <p
            class="quote-page__text"
          >
            Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
          </p>
        </section>
      </main>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <div
        class="header__content"
      >
        <a
          data-testid="logo-link"
          href="/"
        >
          <span
            class="header__logo"
          >
            math magicians
          </span>
        </a>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li>
              <a
                class="header__nav-link"
                href="/"
              >
                home
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/calculator"
              >
                calculator
              </a>
            </li>
            <li>
              <a
                class="header__nav-link"
                href="/quote"
              >
                quote
              </a>
            </li>
          </ul>
        </nav>
        <div
          class="header__hamburger-container"
        >
          <div
            class="hamburger-menu "
          >
            <button
              aria-label="Open hamburger menu"
              class="hamburger-btn "
              type="button"
            >
              <span
                class="hamburger-btn__icon"
              >
                 
              </span>
            </button>
            <div
              class="hamburger-menu__background"
            >
               
            </div>
            <nav
              class="hamburger-menu__nav"
            >
              <ul
                class="hamburger-menu__nav-list"
              >
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/"
                  >
                    <span>
                      01
                    </span>
                    home
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/calculator"
                  >
                    <span>
                      02
                    </span>
                    calculator
                  </a>
                </li>
                <li>
                  <a
                    class="hamburger-menu__link"
                    href="/quote"
                  >
                    <span>
                      03
                    </span>
                    quote
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </header>
    <main
      class="main"
    >
      <section
        class="quote-page"
      >
        <p
          class="quote-page__text"
        >
          Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
        </p>
      </section>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
